care/users/api/serializers/user.py

```diff
@@ -335,6 +335,7 @@ class Meta:
             "pf_auth",
             "read_profile_picture_url",
             "user_flags",
+            "last_login",
```

Comment:
The field has to be in fields to be included in serialized output, no? Changing it to read_only_fields makes it inaccessible via the API. As for modification, since the field is managed by Django, if frontend were to pass data for last_login, Django should ignore it.

Reply:
It wasn't accessible, likely due to inheritance from SignUpSerializer. I can add last_login to both fields and read_only_fields instead.